site/src/component/Report/Reports.tsx

```diff
@@ -56,30 +80,19 @@ const Reports: FC = () => {
     setData(data.filter((review) => review.reviewId !== reviewId));
   };
 
-  if (!loaded) {
-    return <p>Loading...</p>;
-  } else if (data.length === 0) {
-    return <p>No reports to display at the moment.</p>;
-  } else {
-    return (
-      <div className="content-wrapper reports-container">
-        <h1>User Review Reports</h1>
-        <p>Denying a review's reports will discard the reports and preserve the review.</p>
-        <p>Accepting a review's reports will discard the reports and the review.</p>
-        {data.map((review) => {
-          return (
-            <ReportGroup
-              key={review.reviewId}
-              reviewId={review.reviewId}
-              reports={review.reports}
-              onAccept={() => acceptReports(review.reviewId)}
-              onDeny={() => denyReports(review.reviewId)}
-            />
-          );
-        })}
-      </div>
-    );
-  }
+  return (
+    <div className="content-wrapper reports-container">
+      <h1>User Review Reports</h1>
+      <p>
+        Accepting a report will <b>delete</b> the review. Ignoring a report will <b>preserve</b> the review.
+      </p>
+      {reportsLoading ? (
+        <p>Loading...</p>
+      ) : (
```

Comment:
iirc the Loading spinner PR is not fully merged yet? because if it is, I'd say to use that instead of the text "Loading..."